packages/bot-web-ui/src/stores/transactions-store.ts

```diff
@@ -68,11 +68,38 @@ export default class TransactionsStore {
         );
         const statistics = trxs.reduce(
             (stats, { data }) => {
-                const { profit = 0, is_completed = false, payout = 0, buy_price = 0 } = data as TContractInfo;
+                const {
+                    profit = 0,
+                    is_completed = false,
+                    payout = 0,
+                    buy_price = 0,
+                    bid_price = 0,
+                } = data as TContractInfo;
                 if (is_completed) {
                     if (profit > 0) {
                         stats.won_contracts += 1;
                         stats.total_payout += payout;
+                    }
+                    if (bid_price > 0) {
+                        if (window.Blockly.derivWorkspace) {
+                            const trade_options_blocks = window.Blockly.derivWorkspace
+                                .getAllBlocks()
+                                .filter(
+                                    b =>
+                                        b.type === 'trade_definition_multiplier' ||
+                                        b.type === 'trade_definition_accumulator'
+                                );
+                            trade_options_blocks.forEach((trade_options_block: { selected_trade_type: string }) => {
+                                if (
+                                    trade_options_block.selected_trade_type === 'accumulator' ||
+                                    trade_options_block.selected_trade_type === 'multiplier'
+                                ) {
+                                    if (stats.total_payout !== bid_price && profit >= 0 && payout !== bid_price) {
+                                        stats.total_payout += bid_price;
+                                    }
+                                }
+                            });
```

Comment:
@maryia-matskevich-deriv we don't need to make all these changes. Let's get on a huddle with @rupato-deriv . We just need to take the bid_price out from the API response and use it like this: stats.total_payout += payout ?? bid_price ?? 0

We do not need to make all these changes.

Reply:
yes, we've checked, make sense, we can make the solution much shorter, thank you guys🙏